config/initializers/cors.rb

```diff
@@ -1,5 +1,5 @@
 Barong::App.define do |config|
   config.set(:api_cors_origins, '*')
-  config.set(:api_cors_max_age, '3600', type: :integer)
+  config.set(:api_cors_max_age, '3600')
```

Comment:
double check you seem to fix this twice with the to_i
shouldn't it be integer again ?